packages/sanity/src/core/studio/components/navbar/presence/PresenceMenuItem.tsx

```diff
@@ -73,26 +72,31 @@ export const PresenceMenuItem = memo(function PresenceMenuItem(props: PresenceLi
       as={lastActiveLocation ? LinkComponent : 'div'}
       data-as={lastActiveLocation ? 'a' : 'div'}
       onFocus={handleFocus}
-      padding={4}
+      paddingY={hasLink ? 4 : 3}
+      paddingLeft={4}
       paddingRight={3}
       ref={setMenuItemElement}
     >
       <Flex align="center">
         <AvatarCard>
-          <UserAvatar size={1} key={presence.user.id} user={presence.user} />
+          <UserAvatar
+            size={1}
+            key={presence.user.id}
+            user={presence.user}
+            status={hasLink ? 'editing' : 'inactive'}
+          />
         </AvatarCard>
 
         <Box flex={1} marginLeft={4}>
           <Text textOverflow="ellipsis">{presence.user.displayName}</Text>
+          {!hasLink && (
+            <Stack paddingTop={2}>
+              <Text size={0} muted textOverflow="ellipsis">
+                Not in a document
+              </Text>
+            </Stack>
+          )}
```

Comment:
Nitpick (feel free to ignore): In this situation I would wrap the display name and subtitle with `Stack` to create the vertical space. It is arguably a more correct use of `Stack` in this situation. :) 

```tsx
<Stack space={2}>
  <Text textOverflow="ellipsis">{presence.user.displayName}</Text>

  {!hasLink && (
    <Text size={0} muted textOverflow="ellipsis">
      Not in a document
    </Text>
  )}
</Stack>
```

Reply:
Good catch! Fixed this now :)